### ISSUE TITLE
Mac empty when run with real device iOS 15.

### ISSUE THREAD

**Issue by ttrong** [mavris/MMLanScan#65]
When I run it with simulator, it return Mac address but it return "02:00:00:00:00:00" with real device iOS 15.

lanScanDidFindNewDevice: 192.168.31.1 - 02:00:00:00:00:00"
"lanScanDidFindNewDevice: 192.168.31.48 - 02:00:00:00:00:00"
"lanScanDidFindNewDevice: 192.168.31.64 - 02:00:00:00:00:00"
"lanScanDidFindNewDevice: 192.168.31.55 - 02:00:00:00:00:00"

**Comment by nandishyb** [comment]
Have you got any update on this issue ?

**Comment by afinainflowers** [comment]
Same issue for me. Also hostnames are always nill.

**Comment by riteshshah5432** [comment]
+ Same issue

**Comment by beeper99** [comment]
Yes, it won't return MAC on modern iOS devices, this was limited by apple after iOS 7.
Simulator is not limited as running on a MacOS device.
